api/lib/domain/models/CertificationAssessment.js

```diff
@@ -45,6 +45,12 @@ class CertificationAssessment {
     return _.find(this.certificationChallenges, { challengeId }) || null;
   }
 
+  neutralizeChallengeByRecId(recId) {
+    const challengeToBeNeutralized = this.certificationChallenges.find((challenge) => challenge.challengeId === recId);
```

Comment:
suggestion: on fait peut faire comme au dessus

```suggestion
    const challengeToBeNeutralized = _.find(this.certificationChallenges, { challengeId: recId })
```